Demos/src/Routing.cpp

```diff
@@ -741,6 +751,13 @@ int main(int argc, char* argv[])
                       "penalty-max",
                       "Maximum junction penalty, time [s]. Default "s + std::to_string(duration_cast<seconds>(args.maxPenalty).count()));
 
+
+  argParser.AddOption(osmscout::CmdLineGeoCoordOption([&args](const osmscout::GeoCoord& value) {
+                        args.via=value;
```

Comment:
It would be great to accept --via argument multiple times, so make args.via vector of locations

Reply:
Yes I agree but I don’t know how to code it

Reply:
(Without checking the details...). The Command Line Argument Parser does not hold state to error on arguments called multiple times. In this case it should be just as easy as making args.via a list and do a ...push_back() in the callback implementation!?

Reply:
Yes it works, thank you ! I made a pull request with that